### PR TITLE
animate-ambient-light

### PR DESCRIPTION
**Pull request by syedabdullahali**

### CODE REVIEW

File: style1.css [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (Abbas4tech) [inline style1.css:66]: Remove the empty space here @syedabdullahali

File: style1.css [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+#ball1{
+   height: 200px;
+   width:200px;
+   border-radius: 50%;
+   background-color:fuchsia;
+   position: relative;
+   top: 40px;
+   animation-name: king1;
+   animation-duration: 4s;
+   animation-iteration-count: infinite;
```
Comment (Abbas4tech) [inline style1.css:76]: Use short hand @syedabdullahali

**Review by Tauqeer-Ahmed-99 — CHANGES_REQUESTED** [review]
Group similar properties@syedabdullahali

File: style1.css [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+#ball2{
+    height: 200px;
+    width:200px;
+    border-radius: 50%;
+    background-color:rgb(9, 73, 248) ;
+    position: relative;
+    top: 40px;
+    animation-name: example;
+    animation-duration: 4s;
+    animation-iteration-count: infinite;
```
Comment (Abbas4tech) [inline style1.css:98]: Here too @syedabdullahali

File: style1.css [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+#ball3{
+    height: 200px;
+    width:200px;
+    border-radius: 50%;
+    background-color:rgb(248, 3, 215);
+    position: relative;
+    top: 40px;
+    animation-name: king;
+    animation-duration: 4s;
+    animation-iteration-count: infinite;
```
Comment (Abbas4tech) [inline style1.css:122]: Use short hand @syedabdullahali

**Review by Abbas4tech — CHANGES_REQUESTED** [review]

**Review by Tanvir04khan — APPROVED** [review]
Group similar properties

File: style1.css (hunk 1 above)
Comment (syedabdullahali) [reply style1.css]: done

File: style1.css (hunk 2 above)
Comment (syedabdullahali) [reply style1.css:62]: Thank u for this but this was good for understand during coding

File: style1.css (hunk 3 above)
Comment (syedabdullahali) [reply style1.css]: Done

File: style1.css (hunk 4 above)
Comment (syedabdullahali) [reply style1.css]: done

**Comment by syedabdullahali** [comment]
> Group similar properties

**Comment by syedabdullahali** [comment]
> Group similar properties@syedabdullahali

ok

**Review by mujeeb100 — APPROVED** [review]

**Pull request merged**